Index: .
===================================================================
--- .	(revision 9735)
+++ .	(working copy)

Property changes on: .
___________________________________________________________________
Added: svn:ignore
## -0,0 +1 ##
+target
Index: doc/Syntax
===================================================================
--- doc/Syntax	(revision 9735)
+++ doc/Syntax	(working copy)

Property changes on: doc/Syntax
___________________________________________________________________
Added: svn:ignore
## -0,0 +1,2 ##
+IML.synctex.gz
+IML.log
Index: src/SILAST
===================================================================
--- src/SILAST	(revision 9735)
+++ src/SILAST	(working copy)

Property changes on: src/SILAST
___________________________________________________________________
Modified: svn:ignore
## -2,3 +2,5 ##
 .classpath
 .project
 .settings
+bin
+target
Index: src/SILAST/build.sbt
===================================================================
--- src/SILAST/build.sbt	(revision 0)
+++ src/SILAST/build.sbt	(working copy)
@@ -0,0 +1,11 @@
+// SIL AST
+
+name := "silast"
+
+version := "0.1"
+
+scalaVersion := "2.9.1"
+
+scalaSource <<= baseDirectory(_ / "src")
+
+scalaSource in Compile <<= scalaSource
\ No newline at end of file
Index: src/SILAST/project
===================================================================
--- src/SILAST/project	(revision 0)
+++ src/SILAST/project	(working copy)

Property changes on: src/SILAST/project
___________________________________________________________________
Added: svn:ignore
## -0,0 +1 ##
+target
Index: src/SILAST/project/build.properties
===================================================================
--- src/SILAST/project/build.properties	(revision 0)
+++ src/SILAST/project/build.properties	(working copy)
@@ -0,0 +1 @@
+sbt.version=0.11.2
\ No newline at end of file
Index: src/SILAST/sbt-launch.jar
===================================================================
Cannot display: file marked as a binary type.
svn:mime-type = application/octet-stream
Index: src/SILAST/sbt-launch.jar
===================================================================
--- src/SILAST/sbt-launch.jar	(revision 0)
+++ src/SILAST/sbt-launch.jar	(working copy)

Property changes on: src/SILAST/sbt-launch.jar
___________________________________________________________________
Added: svn:mime-type
## -0,0 +1 ##
+application/octet-stream
Index: src/SILAST/sbt.ps1
===================================================================
Cannot display: file marked as a binary type.
svn:mime-type = application/octet-stream
Index: src/SILAST/sbt.ps1
===================================================================
--- src/SILAST/sbt.ps1	(revision 0)
+++ src/SILAST/sbt.ps1	(working copy)

Property changes on: src/SILAST/sbt.ps1
___________________________________________________________________
Added: svn:mime-type
## -0,0 +1 ##
+application/octet-stream
